testData/completion/qualifiedbyname/MappingQualifiedByNameWithAllPossibleVisibilities.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at https://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.complex;
+
+import org.example.dto.Car;
+import org.example.dto.CarDto;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.ap.test.complex.SamePackageMapper;
+import org.mapstruct.helper.qualifiedbyname.external.ExternalMapper;
+
+@Mapper(uses = { ExternalMapper.class, SamePackageMapper.class })
+public interface CarMapper {
+
+    @Mapping(target = "make", qualifiedByName = "<caret>")
+    CarDto carToCarDto(Car car);
+
+    @Named("internalModifierPackagePrivate")
+    String internalModifierPackagePrivate(String value) {
```

Comment:
This is not an `internalModifier` this is public, due to the fact that `CarMapper` is an interface and not a class